### DISCUSSION TITLE
Exceptional Control Flow | Jae’s Blog

### DISCUSSION THREAD

**Discussion by giscus** [jaenam615/jaenam615.github.io#7] [Announcements]
# Exceptional Control Flow | Jae’s Blog

Hydejack is a boutique Jekyll theme for hackers, nerds, and academics, with a focus on personal sites that are meant to impress.

https://jaenam615.github.io/computer_science,/os/2024-02-08-exceptional_control_flow/

**Comment by JeongJongMun** [comment]
정리가 참 잘 되어있네요 덕분에 많이 배우고 갑니다
